docs/README.md

```diff
@@ -257,3 +257,189 @@ To disable that behavior, use the `compactMode` parameter in JSON-RPC request:
     }
 }
 ```
+
+# FAQ
+
+## What is the difference between Sources and SceneItems?
+
+One thing to keep in mind in OBS is that sources and scene items are different things.
+A source exists outside the context of a scene. Muted is a property of a source. 
+A scene item is essentially an instance of a source in a scene. 
+The same underlying source can be in multiple scenes as a sceneitem. 
+Visibility is an attribute on a scene item. 
+So the same source can be visible in one scene and invisible in another for example.
```

Comment:
This looks familiar 😆

Reply:
Yeah, that is the coarse theft of my laziness =)